### ISSUE TITLE
Warn before encrypting/signing when composing HTML mails

### ISSUE THREAD

**Issue by stefan0xC** [Zimbra-Community/pgp-zimlet#233]
Pressing the encrypt or sign button switches the mode of the editor from HTML to plain-text without warning. A warning (like the one switching from HTML to plain-text format manually) would be nice and/or cancelling should revert the original formatting.

Also while the mode of the editor changes, the editor options don't reflect that change. To revert the original message, I have to first change from HTML to plain-text mode, accept the warning and only then I am able to switch back to HTML where I can restore the original formatting.

**Comment by barrydegraaff** [comment]
yes, it could be dealt with nicer, but currently no dev work is scheduled on this.
